media/js/src/filters/AssetFilter.jsx

```diff
@@ -252,9 +252,10 @@ export default class AssetFilter extends React.Component {
                             </div>
                         </div>
                         <div className="form-group col-md-2">
-                            <label htmlFor="react-select-2-input">Owner</label>
+                            <label htmlFor="ownerFilter">Owner</label>
                             <Select
                                 id="owner-filter"
+                                inputId='ownerFilter'
```

Comment:
html ID's must be unique on the page - so just be aware of that when implementing these sorts of things. It may be best to generate a random string or number here, just to make sure there are no conflicts with something else on the page, and do something like:
```
const uniqueId = 'abcdef'; // generate this randomly, though
const ownerFilterId = `ownerFilter-${uniqueId}`;
```
Then in the template:
```
<label htmlFor={ownerFilterId}>Owner</label>
<Select
    inputId={ownerFilterId}
```

Reply:
Honestly, all that may not be necessary though. Your convention here looks good. We can roll that out and extend on it if needed or we see any issues.